test/integration/statistics_announcement_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class StatisticsAnnouncementTest < ActionDispatch::IntegrationTest
+
+  test "official statistics are rendered correctly" do
+    setup_and_visit_content_item('official_statistics')
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+    assert_has_component_metadata_pair("Release date", "20 January 2016 9:30am (confirmed)")
+  end
+
+  test "national statistics are rendered correctly" do
+    setup_and_visit_content_item('national_statistics')
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+    assert page.has_css?('.national-statistics-logo img')
+    assert_has_component_metadata_pair("Release date", "January 2016 (provisional)")
```

Comment:
I've changed the test names, which I think is enough for now. I think we can improve the readability of these kind of tests as we add more.